libmamba/include/mamba/core/context.hpp

```diff
@@ -118,11 +118,59 @@ namespace mamba
     {
     public:
 
-        std::string caller_version = "";
-        std::string conda_version = "3.8.0";
-        std::string current_command = "mamba";
-        std::string custom_banner = "";
-        bool is_micromamba = false;
+        // TODO separate later between configurables and internal variables
+        struct RemoteFetchInfo
+        {
+            // ssl_verify can be either an empty string (regular SSL verification),
+            // the string "<false>" to indicate no SSL verification, or a path to
+            // a directory with cert files, or a cert file.
+            std::string ssl_verify{ "" };
+            bool ssl_no_revoke{ false };
+            bool curl_initialized{ false };  // non configurable, used in fetch only
+
+            std::string user_agent{ "mamba/" LIBMAMBA_VERSION_STRING };
+
+            int connect_timeout_secs{ 10 };
+            // int read_timeout_secs { 60 };
+            int retry_timeout{ 2 };  // seconds
+            int retry_backoff{ 3 };  // retry_timeout * retry_backoff
+            int max_retries{ 3 };    // max number of retries
+        };
+
+        struct OutputInfo  // Configurable, TODO change name of struct?
```

Comment:
Should we use names for structs which would reflect configurability like `InternalDesignInfo` and `ConfigSrcInfo`?

Reply:
I don't think so, Configurability is something that can evolve in the time, and it will be hard to keep things synchronized. Besides, once the configuration has been load, we don't really care where the values come from.